app/models/promotional_feature_item.rb

```diff
@@ -6,8 +6,15 @@ class PromotionalFeatureItem < ApplicationRecord
   validates :summary, presence: true, length: { maximum: 500 }
   validates_with ImageValidator, method: :image, size: [960, 640], if: :image_changed?
   validates :title_url, uri: true, allow_blank: true
+  validate :image_or_youtube_url_is_present
 
   accepts_nested_attributes_for :links, allow_destroy: true, reject_if: ->(attributes) { attributes["url"].blank? }
 
   mount_uploader :image, ImageUploader
+
+private
+
+  def image_or_youtube_url_is_present
+    errors.add(:base, "Upload an image or add a Youtube URL") if image.blank? && youtube_video_url.blank?
```

Comment:
I did think about this, but thought that work arounds would probably be a bit overkill since this a sparsely touched feature and it'll likely only be a few days before it goes fully live.

Reply:
Yup makes sense.